docs/user/documents.rst

```diff
@@ -78,11 +78,11 @@ a document like so::
     # or
 
     with open('foobar.docx', 'rb') as f:
-        source_stream = StringIO(f.read())
+        source_stream = BytesIO(f.read())
```

Comment:
Hmm, interesting. I'm sure this is an artifact of this documentation being written during the Python 2 era, when `StringIO` actually meant "read bytes".

I'll put it on the shortlist and fix it up next time I'm in there.